sql/src/main/java/org/apache/druid/sql/calcite/rule/GroupByRules.java

```diff
@@ -69,6 +71,17 @@ public static Aggregation translateAggregateCall(
       return null;
     }
 
+    if (plannerContext.getPlannerConfig().isUseApproximateCountDistinct() && call.isDistinct()) {
```

Comment:
at this point we should not care about `isUseApproximateDistinct` as those supposed to be rewritten already